dds/DCPS/XTypes/TypeLookupService.cpp

```diff
@@ -1050,6 +1077,43 @@ void TypeLookupService::remove_guid_from_dynamic_map(const DCPS::GUID_t& guid)
 }
 #endif
 
+bool TypeLookupService::parse_extensibility(const DCPS::String& x, DDS::ExtensibilityKind& d)
+{
+  if (x == "FINAL") {
```

Comment:
Should this allow case-insensitive matching?  Since the annotation is `@final`, users may be thinking of it in lower case.